client/src/views/CreateCopyrightApplication.vue

```diff
@@ -664,15 +668,31 @@ export default {
         yearCompleted: {
           required,
           minValue: minValue(minYearCompleted),
-          maxValue: maxValue(maxYearCompleted)
+          maxValue: maxValue(thisYear)
         },
         publicationDate: {
           mmddyyyy: mmddyyyy,
-          within: (publicatonDate) => {
-            if (!empty(publicatonDate) && !empty(this.form.yearCompleted)) {
-              let publicatonYear = parseInt(publicatonDate.substring(4, 8))
-              return ((publicatonYear <= this.form.yearCompleted) &&
-                (publicatonYear >= this.form.yearCompleted - 95))
+          after: (publicationDate) => {
+            if (!empty(publicationDate)) {
+              let publicationMonth = parseInt(publicationDate.substring(0, 2))
```

Comment:
I think this can be simplified with the user of momentJs, see ValidatonHelpers.js in the util directory

Reply:
Also, why not just use the existing within method?

Reply:
@lipsotiko Looks great, thanks!